### PR TITLE
added metric units (feet to meter)

### PR DESCRIPTION
**Pull request by zeno4ever**
Added meter tekst to distance selection.

### CODE REVIEW

**Review by beshaya — APPROVED** [review]
Thanks for fixing our US-centrism!

**Comment by blanchardjeremy** [comment]
Looks good to me 👍

**Review by blanchardjeremy — APPROVED** [review]
👍

File: src/data/data.ts [hunk 1]
```diff
@@ -26,9 +26,9 @@ export const Setting: { [key: string]: FormValue } = {
 }
 export const Distance: { [key: string]: FormValue } = {
   intimate: formValue('Cuddling / intimate', 2),
-  normal: formValue('Normal socializing (~3 feet apart)', 1),
-  sixFt: formValue('6+ feet apart', 0.5),
-  tenFt: formValue('10+ feet apart', 0.25),
+  normal: formValue('Normal socializing (~3 feet/ ~1 meter apart)', 1),
+  sixFt: formValue('6+ feet / ~1.80 meter apart', 0.5),
```
[the remark this answers is not in the document]
Comment (blanchardjeremy) [reply src/data/data.ts]: Oops. This is out of date. Please proceed.

File: src/data/data.ts [hunk 2]
```diff
@@ -26,9 +26,9 @@ export const Setting: { [key: string]: FormValue } = {
...
+  tenFt: formValue('10+ feet / ~3 meter apart', 0.25),
```
[the remark this answers is not in the document]
Comment (blanchardjeremy) [reply src/data/data.ts]: Oops. This is out of date. Please proceed.

**Pull request merged**